gimie/cli.py

```diff
@@ -32,9 +34,7 @@ def cli():
     pass
 
 
-class SerializationFormat(str, Enum):
-    """Enumeration of valid RDF serialization formats for project graphs"""
-
+class FormatChoice(str, Enum):
```

Comment:
could we explicit the name further for it to be standalone: LinkedDataFormatChoice or LDFormatChoice ?

Reply:
Tried to compromise between clarity and conciseness in [a6fb7e7](https://github.com/SDSC-ORD/gimie/pull/97/commits/a6fb7e71cab547e9a537d84dfc3bd0d327141a71)